main.go

```diff
@@ -388,6 +402,35 @@ type server struct {
 	ServerFeatures []string `json:"server_features,omitempty"`
 }
 
+func generateServerConfigsFromInputs(serverUris []string, in configInput) []server {
```

Comment:
I understand this is future proof in the sense that it will handle the case where we want to support multiple server configs. But we dont have that need today and we don't foresee it in the near future. So, I would recommend changing this function to be `func generateServerConfigsFromInputs(serverUri string, in configInput) []server { ...}`.

See: https://engdoc.corp.google.com/eng/doc/tott/episodes/438.md?cl=head

Reply:
:+1: makes sense.. Im updating this as well